hetzner/README-cloud.md

```diff
@@ -91,6 +96,42 @@ It's used so Ansible can connect to the server without requiring password.
 
 # The VM
 
+## All-in-one creation
+
+2 new playbooks have been created to aggregate the different operations executed by the different playbooks
```

Comment:
`2 new playbooks have been created to aggregate the different operations executed by the different playbooks` --> `2 new playbooks have been created to aggregate the operations executed by the different playbooks`